src/Microsoft.Data.Analysis/ColumnArithmeticTemplate.cs

```diff
@@ -0,0 +1 @@
+﻿
```

Comment:
This should be deleted.

Reply:
Damn, this keeps creeping in. I just deleted the include entry from the .csproj.

Reply:
Take these two lines out:

https://github.com/dotnet/corefxlab/blob/b23587dd66e3737b443b7b85c41b2571e884535d/src/Microsoft.Data.Analysis/Microsoft.Data.Analysis.csproj#L75-L76